tools/integration_tests/util/operations/file_operations.go

```diff
@@ -141,3 +143,45 @@ func CloseFile(file *os.File) {
 		log.Printf("error in closing: %v", err)
 	}
 }
+
+func ReadFileSequentially(filePath string, numberOfBytes int64) (content []byte, err error) {
```

Comment:
we shouldn't take numberOfBytes as argument as we want to read complete file. Also, we should take chunkSize as argument.

Reply:
Fixed